https://bugs.gentoo.org/show_bug.cgi?id=346577

# HG changeset patch
# User Vedran MiletiÄ‡ <rivanvx@gmail.com>
# Date 1281186298 -7200
# Node ID 5ac9c9444ac7056cb450866bf14015910bd28395
# Parent  306d55f66f5b6ce1a3bda0c2dfb006960d2bccee
Fix #53. Thanks for catching that, Stanislaw Polak.

diff -r 306d55f66f5b -r 5ac9c9444ac7 base/translator/dicts/translator-numbers-dictionary/translator-numbers-dictionary-Polish.dict
--- a/base/translator/dicts/translator-numbers-dictionary/translator-numbers-dictionary-Polish.dict	Mon Jul 12 19:49:46 2010 +0200
+++ b/base/translator/dicts/translator-numbers-dictionary/translator-numbers-dictionary-Polish.dict	Sat Aug 07 15:04:58 2010 +0200
@@ -10,7 +10,7 @@
 \providetranslation{seven}{siedem}
 \providetranslation{eight}{osiem}
 \providetranslation{nine}{dziewi\k{e}\'{c}}
-\providetranslation{ten}{dziesi\{e}\'{c}}
+\providetranslation{ten}{dziesi\k{e}\'{c}}
 \providetranslation{eleven}{jedena\'{s}cie}
 \providetranslation{twelve}{dwana\'{s}cie}
 
@@ -23,7 +23,7 @@
 \providetranslation{Seven}{Siedem}
 \providetranslation{Eight}{Osiem}
 \providetranslation{Nine}{Dziewi\k{e}\'{c}}
-\providetranslation{Ten}{Dziesi\{e}\'{c}}
+\providetranslation{Ten}{Dziesi\k{e}\'{c}}
 \providetranslation{Eleven}{Jedena\'{s}cie}
 \providetranslation{Twelve}{Dwana\'{s}cie}
 
